README.md

```diff
@@ -132,6 +133,34 @@ for logger_name in ('snowflake.snowpark', 'snowflake.connector'):
     logger.addHandler(ch)
 ```
 
+## Reading and writing to pandas DataFrame
+
+Snowpark DataFrame API supports reading from and writing to a pandas DataFrame via the [to_pandas][to_pandas] and [write_pandas][write_pandas] commands. 
+
+To use these operations, ensure that pandas is installed in the same environment. You can install pandas alongside Snowpark Python by executing the following command:
+```bash
+pip install "snowflake-snowpark-python[pandas]"
+```
+Once pandas is installed, you can convert between a Snowpark DataFrame and pandas DataFrame as follows: 
+```python
+df = session.create_dataframe([[1, 2], [3, 4]], schema=["a", "b"])
+# Convert Snowpark DataFrame to pandas DataFrame
+pandas_df = df.to_pandas() 
+# Write pandas DataFrame to a Snowflake table and return Snowpark DataFrame
+snowpark_df = session.write_pandas(pandas_df, "new_table", auto_create_table=True)
+```
+
+Snowpark pandas API also supports writing to pandas: 
+```python
+import modin.pandas as pd
```

Comment:
i think we can remove the import here like the python, because to have a complete example, it also needs the session. Since we already have an example above I think we should be fine here

Reply:
I removed the snowpark import but kept the modin one just so that it is clear this is a Snowpark pandas dataframe, since the API signature is the same as pandas.